From 8a2933404c03f2e1103dc64bac3497fa4bd5502c Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Fri, 15 Apr 2022 00:49:58 +0800
Subject: [PATCH 3/3] Revert "drm/vrr: Set VRR capable prop only if it is
 attached to connector"

This reverts commit 0ba557d330946c23559aaea2d51ea649fdeca98a.
---
 drivers/gpu/drm/drm_connector.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/gpu/drm/drm_connector.c b/drivers/gpu/drm/drm_connector.c
index 5163433ac561b..717c4e7271b04 100644
--- a/drivers/gpu/drm/drm_connector.c
+++ b/drivers/gpu/drm/drm_connector.c
@@ -2155,9 +2155,6 @@ EXPORT_SYMBOL(drm_connector_attach_max_bpc_property);
 void drm_connector_set_vrr_capable_property(
 		struct drm_connector *connector, bool capable)
 {
-	if (!connector->vrr_capable_property)
-		return;
-
 	drm_object_property_set_value(&connector->base,
 				      connector->vrr_capable_property,
 				      capable);
-- 
2.27.0

